core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -130,11 +130,11 @@ public ConnectorToSqlExpressionTranslator(Session session, PlannerContext planne
             }
 
             if (LIKE_PATTERN_FUNCTION_NAME.equals(call.getFunctionName())) {
-                switch (call.getArguments().size()) {
+                switch (call.getArgumentsCount()) {
                     case 2:
-                        return translateLike(call.getArguments().get(0), call.getArguments().get(1), Optional.empty());
+                        return translateLike(call.getArgument(0), call.getArgument(1), Optional.empty());
```

Comment:
We don't need new methods (getArgumentsCount, getArgument) for this.
It's enough to capture `call.getArguments()` as a local variable

Reply:
We have similiar methods in BoundSignature (`getArgumentType(index)`, `getArity()`). Should we remove them as well?